src/material/list/list.md

```diff
@@ -24,18 +24,18 @@ Simple navigation lists can use the `mat-list-item` attribute on anchor tag elem
 
 ```html
 <mat-nav-list>
-   <a mat-list-item href="..." *ngFor="let link of links"> {{ link }} </a>
+  <a mat-list-item href="..." *ngFor="let link of links">{{ link }}</a>
 </mat-nav-list>
 ```
 
-For more complex navigation lists (e.g. with more than one target per item), wrap the anchor 
+For more complex navigation lists (e.g. with more than one target per item), wrap the anchor
 element in an `<mat-list-item>`.
 
 ```html
 <mat-nav-list>
   <mat-list-item *ngFor="let link of links">
-     <a matLine href="...">{{ link }}</a>
-     <button mat-icon-button (click)="showInfo(link)">
+     <a matListItemLine href="...">{{ link }}</a>
```

Comment:
I wonder if we need to clarify the API a bit more, of should that be a follow-up?

Every list item must have a title — it cannot use lines without an explicit title. Maybe also http://go/angular-mdc/migration#list is helpful.

```suggestion
     <a matListItemTitle href="...">{{ link }}</a>
```

Reply:
Changed the example to have a title. Do we have validation at runtime that it has a title? If it does, I don't think that we need to call it out specifically.

Reply:
Yeah, we have some sanity checks in: https://github.com/angular/components/blob/main/src/material/list/list-base.ts#L323. I think it would be still nice at some point to talk about the new API more in detail — like the support for text wrapping and the explicit `lines` attribute.